packages/api/config/lambdas.yml

```diff
@@ -4,6 +4,7 @@ sqs2sf:
   memory: 512
   source: 'node_modules/@cumulus/api/dist/'
 
+
 executeMigrations:
```

Comment:
What did we decide on this one?

Reply:
We should chat about this one, it's tied up in the async operations/Bulk Delete work that isn't completed.

Reply:
I've posed a question about this ticket's AC to @bilts, we can always add the config option if it's determined my assertion is incorrect prior to closing 817.